bigquery_etl/newtab_merino/__init__.py

```diff
@@ -0,0 +1,107 @@
+"""Export data from BigQuery to GCS."""
+
+import json
+from datetime import datetime, timedelta, timezone
+
+import rich_click as click
+from google.cloud import storage  # type: ignore
```

Comment:
[known issue](https://github.com/googleapis/python-storage/issues/393) w/ google cloud storage and type hints